designcompose/src/main/java/com/android/designcompose/squoosh/SquooshVariantTransition.kt

```diff
@@ -59,12 +59,21 @@ import com.android.designcompose.serdegen.View
 
 interface AnimationTransition {
     fun animationSpec(): AnimationSpec<Float>
+
+    fun delayMs(): Int
 }
 
-class SmartAnimateTransition(val transition: AnimationSpec<Float>) : AnimationTransition {
+class SmartAnimateTransition(
```

Comment:
Since this is public API it should have some documentation.

Reply:
Added some comments and edited the animation documentation